pkg/roachprod/vm/azure/utils.go

```diff
@@ -22,11 +22,9 @@ import (
 // created from /mnt/data<disknum> to the mount point.
 // azureStartupArgs specifies template arguments for the setup template.
 type azureStartupArgs struct {
-	RemoteUser      string // The uname for /data* directories.
-	AttachedDiskLun *int   // Use attached disk, with specified LUN; Use local ssd if nil.
-	// TODO(DarrylWong): In the future, when all tests are run on Ubuntu 22.04, we can remove this check and default true.
-	// See: https://github.com/cockroachdb/cockroach/issues/112112
-	IsUbuntu22 bool // Allow RSA SHA1 to be used and create tcpdump symlink.
```

Comment:
This is still present in the startup script, should we remove it / will it cause issues?

See below:
```
{{ if .IsUbuntu22 }}
sudo sh -c 'echo "PubkeyAcceptedAlgorithms +ssh-rsa" >> /etc/ssh/sshd_config'
{{ end }}
```

Reply:
Good catch! I wasn't entirely sure if we had any roachtests on this branch using an older version of Ubuntu. Indeed, we do,

```
pkg/cmd/roachtest/tests/ruby_pg.go:		Cluster:          r.MakeClusterSpec(1, spec.UbuntuVersion(vm.FocalFossa)),
pkg/cmd/roachtest/tests/disk_stall.go:			Cluster:             r.MakeClusterSpec(4, spec.ReuseNone(), spec.DisableLocalSSD(), spec.UbuntuVersion(vm.FocalFossa)),
```

Reverting that change.